### PR TITLE
feature: add disable-bridge switch

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
add disable-bridge switch for daemon config
disable-bridge is a daemon feature same as moby.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
Added

### Ⅳ. Describe how to verify it
ci pass

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: network/mode/init.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/alibaba/pouch/daemon/mgr"
 	"github.com/alibaba/pouch/network"
 	"github.com/alibaba/pouch/network/mode/bridge"
```
Comment (HusterWan) [inline network/mode/init.go]: add blank line ?

File: test/daemonv2/daemon.go [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+func (d *Daemon) Stop() error {
+	if !d.IsUp() {
+		return nil
+	}
+
+	if d.Pid != 0 {
+		// kill pouchd and all other process in its group
+		err := syscall.Kill(-d.Pid, syscall.SIGKILL)
+		if err != nil {
+			fmt.Printf("kill pouchd failed, err:%s", err)
```
Comment (fuweid) [inline test/daemonv2/daemon.go]: I think we can use `pkgerrors.Wrap` and return error. Then caller will check or log the error. What do you think?

File: test/daemonv2/daemon.go [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+// Start starts pouch daemon with config file, instead of running with arguments
+func (d *Daemon) Start() error {
+	fd, err := os.OpenFile(d.ConfigJSON, os.O_CREATE|os.O_RDWR|os.O_TRUNC, 0755)
+	if err != nil {
+		return err
+	}
+
+	if err := json.NewEncoder(fd).Encode(&d.Config); err != nil {
```
Comment (fuweid) [inline test/daemonv2/daemon.go]: add `fd.Close()` here?

File: test/daemonv2/daemon.go [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+	// start pouchd daemon
+	cmd := exec.Command(d.Bin, "--config-file", d.ConfigJSON)
+
+	d.LogFile, err = os.Create(d.LogPath)
+	if err != nil {
+		return fmt.Errorf("failed to create log file %s, err %s", d.LogPath, err)
+	}
+	// Must not close the outfile
+	//defer outfile.Close()
```
Comment (fuweid) [inline test/daemonv2/daemon.go]: remove the comment line?

File: test/daemonv2/daemon.go [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+func (d *Daemon) Restart() error {
+	if err := d.Stop(); err != nil {
+		return err
+	}
+
+	return d.Start()
+}
+
+// Stop stops pouch daemon
+func (d *Daemon) Stop() error {
```
Comment (fuweid) [inline test/daemonv2/daemon.go]: just curiosity, if we stop and restart it, the previous part of log will be missing, right?

could we consider this case?

**Comment by allencloud** [comment]
Could you add more details into the PR description?
What this pr solves? What the reason? What the scenario？ @rudyfly

**Comment by rudyfly** [comment]
@allencloud disable-bridge function same as moby.

File: test/daemonv2/daemon.go (hunk 5 above)
Comment (rudyfly) [reply test/daemonv2/daemon.go]: log has wrote into log file, why it will be missing?

File: test/daemonv2/daemon.go [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+	return &Daemon{
+		Config: daemon.Config{
+			Listen:         []string{Listen},
+			HomeDir:        HomeDir,
+			ContainerdAddr: ContainerdAdd,
+			Pidfile:        Pidfile,
+			Debug:          true,
+		},
+		Bin:        PouchdBin,
+		LogPath:    DaemonLog,
```
Comment (fuweid) [inline test/daemonv2/daemon.go]: I would like to use temp dir as log path and remove it in the cleanup.

**Pull request merged**